lib/ui/profile/components/dao_widget.dart

```diff
@@ -0,0 +1,83 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:hypha_wallet/core/network/models/dao_data_model.dart';
+import 'package:hypha_wallet/design/dividers/hypha_divider.dart';
+import 'package:hypha_wallet/design/hypha_card.dart';
+import 'package:hypha_wallet/design/hypha_colors.dart';
+import 'package:hypha_wallet/design/themes/extensions/theme_extension_provider.dart';
+
+class DaoWidget extends StatelessWidget {
+  final DaoData dao;
+
+  const DaoWidget({
+    super.key,
+    required this.dao,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    // final base64String = dao.settingsLogo;
```

Comment:
@n13  how do we convert this QmV3KmaoqCCXuCDvHzYWS9Jg3RfjrDTQSXK1e7453qfSRS:svg into an svg? 
I think you had some code for this?

Reply:
it's an IPFS link - there's several flutter packages that implement an ipfs bridge.

Reply:
See https://github.com/hypha-dao/hypha_wallet/pull/204#issue-1789124318

Reply:
Looks like I asked ChatGPT - it's good at these things. Code is not complete but chances are it works more or less like that.

Reply:
Ill add this on a follow up PR. No need to clutter this one
we need to create a service for Ipfs stuff now